Cleanup

samkys opened the pull request:
This PR is a cleanup of the microstrain_3dm file. It had various tabs and indentation levels making it very hard to read and follow. Also curly brace location was corrected according to:
http://wiki.ros.org/CppStyleGuide section 6.

Tabs were replaced with spaces. Everything compiles and it was tested with the -25 model and it works with that model.

Commit:
Added static IMU message covariance population via parameters.

Commit:
Cleanup that was forgotten in last commit.

Commit:
Cleaned up indentation levels, removed tabs and replaced with spaces, and updated curly brace locations according to: http://wiki.ros.org/CppStyleGuide section 6.

bsb808 commented:
A quick look at the source looks encouraging.  Thank you for taking the time to add the functionality and help with the maintenance!  

I'll try to find some time to test the driver with a -45 model.

samkys commented:
I was trying to separate out the two pull requests, the cleanup and the parameterization but did them in backwards order. I will close #20 since it is included in this PR anyhow.

samkys commented:
I just came across an interesting case that I cannot test with the -25. Can someone who has a -45 or -35 please test this prior to merging this PR.

1) Set: publish_imu = false
2) Set: publish_odom = true

Does your nav message contain orientation information? 
I think there might be a setting that we are required to have both either to true or fix some things in the driver. (ie set some AHRS settings are required for the filtered data that is being received from the filtered 0x82 packet data set.)

tonybaltovski commented:
@samkys would you mind adding [roslint](http://wiki.ros.org/roslint) test for the formatting?

samkys commented:
@tonybaltovski 
Do you want me to leave roslint in the CMAKE and in the package.xml?
Do you want me to clean all of the messages it throws out?

bsb808 commented:
I ran the test @samkys asked for with a -45 on my desk.  
* publish_imu = false
* publish_filtered_imu = false

The Odometry messages on nav/odom included orientation information, which is what we would want.

Good to go on this end.

tonybaltovski commented:
@samkys that would be ideal to clean-up the package. I can if you just want to add the roslint package.

Commit:
Add roslint and cleaned up files accordingly.

samkys commented:
@tonybaltovski I added roslint and ran it. I cleaned up as much as I could but there were a few lines that I'm not sure what to do with. A lot of the errors it was throwing were too long of lines so I wrapped them around. Take a look and see what you think.

tonybaltovski commented:
Awesome, thanks for that!
